VAMobile/src/store/slices/authSlice.ts

```diff
@@ -737,6 +737,17 @@ const authSlice = createSlice({
     dispatchFinishAuthLogin: (state, action: PayloadAction<AuthFinishLoginPayload>) => {
       const successfulLogin = !action.payload.error
 
+      if (action.payload.error) {
+        return {
+          ...initialAuthState,
+          ...action.payload,
+          webLoginUrl: undefined,
+          initializing: false,
+          successfulLogin: successfulLogin,
+          loggedIn: successfulLogin,
+        }
+      }
+
```

Comment:
it looks like 4/6 of the returns are identical with or without the payload.error. Could we conditionalize the 2/6 instead of the duplications?

Reply:
I'm not saying we have to with this, just interested if we could.

Reply:
yup that can be done